### ISSUE TITLE
Exercises 6.19-6.20 instructions appear to begin with an obsolete assumption

### ISSUE THREAD

**Issue by liamst19** [fullstack-hy2020/fullstack-hy2020.github.io#398]
[Exercise 6.19](https://fullstackopen.com/en/part6/connect#exercises-6-19-6-21) begins with the following assumption:

> The redux store is currently passed to all of the components through props.

Which, if I am not mistaken, is untrue if we've been following the previous exercises that required us to use the newer `useSelector()` hook. It was also stated [in the beginning of the chapter](https://fullstackopen.com/en/part6/connect#using-the-connect-function-to-share-the-redux-store-to-components) that 

> In new applications you should absolutely use the hook-api, but knowing how to use connect is useful when maintaining older projects using redux.

It seems the instructions for the exercises should reflect this and mention that we are to use an older method for legacy purposes, and reiterate that for new projects we should probably be using the `useSelector()` hook-api. 

Also, perhaps it would be helpful for many of the students if a separate project could be provided, so we will not be thoughtlessly overwriting the newer method of state management that we can later reference as notes.

**Comment by Translucent504** [comment]
I had the same problem when working on this exercise, I was wondering if I had skipped some part where we started passing state through props. We had also been using 'react-redux' from the start and now it was being requested to install it again?

I modified the description and made a [pull request](https://github.com/fullstack-hy2020/fullstack-hy2020.github.io/pull/694)

**Comment by railgaadi** [comment]
I came searching for the same issue. I just reached the part and I'm not sure how to proceed. Maybe I will create a clone of the redux-anecdote project and make it use `connect` to share store with components. I personally am going through this tremendous course for hobby and don't see myself having to deal with legacy code. However, a lot of online answers for Redux problems seem to be using `connect` as with most tutorials still assuming usage of class-based components in React. So no harm in giving it a quick go either.
